mlir-tensorrt/tensorrt/include/mlir-tensorrt-dialect/TensorRT/IR/TensorRTOps.td

```diff
@@ -224,6 +224,8 @@ def TensorRT_ActivationOp : TensorRT_Op<"activation", [Pure,
     nvinfer1::IActivationLayer *layer = $net->addActivation(
                                                     *$input,
                                                     *$activationType);
+    if (!layer)
```

Comment:
Leaving a comment open so I can also update `cast` once I merge that (I don't want to edit that PR since it's already passed the pipeline/code review).